pom.xml

```diff
@@ -207,8 +205,35 @@
                     </plugin>
                     <plugin>
                         <groupId>io.fabric8</groupId>
-                        <artifactId>fabric8-maven-plugin</artifactId>
-                        <version>${version.fabric8.maven.plugin}</version>
+                        <artifactId>docker-maven-plugin</artifactId>
+                        <version>${version.docker.maven.plugin}</version>
+                        <configuration>
+                            <images>
+                                <image>
+                                    <name>${docker.name}</name>
+                                    <build>
+                                        <from>java:jre</from>
```

Comment:
`java` was deprecated in Dec 2016: https://hub.docker.com/_/java/. Can you update the image?